code/game/objects/items/stacks/medical.dm

```diff
@@ -511,12 +511,11 @@
 		return BRUTELOSS
 
 	patient.emote("scream")
-	for(var/i in patient.bodyparts)
-		var/obj/item/bodypart/bone = i // fine to just, use these raw, its a meme anyway
-		var/datum/wound/blunt/bone/severe/oof_ouch = new
-		oof_ouch.apply_wound(bone, wound_source = "bone gel")
-		var/datum/wound/blunt/bone/critical/oof_OUCH = new
-		oof_OUCH.apply_wound(bone, wound_source = "bone gel")
+	if (iscarbon(patient))
+		var/mob/living/carbon/carbon_patient = patient
+		for(var/i in carbon_patient.bodyparts)
+			var/obj/item/bodypart/bone = i
+			carbon_patient.cause_wound_of_type_and_severity(WOUND_BLUNT, bone, WOUND_SEVERITY_SEVERE, WOUND_SEVERITY_CRITICAL, wound_source = "bone gel")
```

Comment:
This looks like you are changing how this functions, from applying two wounds of different severity to apply one round of random severity, is that the case?

Reply:
nope

i dont actually know what its doing, because you cant have 2 wounds of the same series on a limn, and it just prioritizes the highest one, so this has the same effect

Reply:
I don't know if that is true
Because in testing it does this
![image](https://github.com/tgstation/tgstation/assets/51863163/e508ba92-4439-4c5c-98bc-948ab6ee811e)